cost-model/src/cost_tracker.rs

```diff
@@ -226,6 +226,7 @@ impl CostTracker {
 
         datapoint_info!(
             "cost_tracker_stats",
+            "is_leader" => is_leader.to_string(),
```

Comment:
didn't realize tags had to be strings. this is probably okay since the allocation pattern is pretty regular (1 per slot)

Reply:
yea, learned that as news too. Thought to transform `true/false` to `1/0` string, but felt little hacky, wondering doing that would make future dev wonder "why".